pkg/api/mlflow/service/run/validators_test.go

```diff
@@ -0,0 +1,329 @@
+package run
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/G-Research/fasttrack/pkg/api/mlflow/api"
+	"github.com/G-Research/fasttrack/pkg/api/mlflow/api/request"
+)
+
+func TestValidateUpdateRunRequest_Ok(t *testing.T) {
+	err := ValidateUpdateRunRequest(&request.UpdateRunRequest{
+		RunID:   "id",
+		RunUUID: "uuid",
+	})
+	assert.Nil(t, err)
+}
+func TestValidateUpdateRunRequest_Error(t *testing.T) {
+	var testData = []struct {
+		name    string
+		error   *api.ErrorResponse
+		request *request.UpdateRunRequest
+	}{
+		{
+			name:    "EmptyRunIDAndRunUUID",
+			error:   api.NewInvalidParameterValueError("Missing value for required parameter 'run_id'"),
+			request: &request.UpdateRunRequest{},
+		},
+	}
+
+	for _, tt := range testData {
+		t.Run(tt.name, func(t *testing.T) {
+			err := ValidateUpdateRunRequest(tt.request)
+			assert.Equal(t, tt.error, err)
+		})
+	}
+}
+
+func TestValidateGetRunRequest_Ok(t *testing.T) {
+	err := ValidateGetRunRequest(&request.GetRunRequest{
+		RunID: "id",
+	})
+	assert.Nil(t, err)
+}
+func TestValidateGetRunRequest_Error(t *testing.T) {
+	var testData = []struct {
+		name    string
+		error   *api.ErrorResponse
+		request *request.GetRunRequest
+	}{
+		{
+			name:    "EmptyRunIDAndRunUUID",
+			error:   api.NewInvalidParameterValueError("Missing value for required parameter 'run_id'"),
+			request: &request.GetRunRequest{},
+		},
+	}
+
+	for _, tt := range testData {
+		t.Run(tt.name, func(t *testing.T) {
+			err := ValidateGetRunRequest(tt.request)
+			assert.Equal(t, tt.error, err)
+		})
+	}
+}
+
+func TestValidateDeleteRunRequest_Ok(t *testing.T) {
+	err := ValidateDeleteRunRequest(&request.DeleteRunRequest{
+		RunID: "id",
+	})
+	assert.Nil(t, err)
+}
+func TestValidateDeleteRunRequest_Error(t *testing.T) {
+	var testData = []struct {
+		name    string
+		error   *api.ErrorResponse
+		request *request.DeleteRunRequest
+	}{
+		{
+			name:    "EmptyRunIDAndRunUUID",
```

Comment:
Remember to change this name once `DeleteRunRequest` does not have `RunUUID` + same for `UpdateRunRequest`

Reply:
for DeleteRunRequest - yes, for UpdateRunRequest - no. UpdateRunRequest still have both parameters.